### PR TITLE
PR JavaScript: Wather App

### PR DESCRIPTION
**Pull request by pastorp3**
This is Project #4 in the JavaScript V1.1 curriculum. The purpose of the project is to create a wather app in which all the content is generated via JavaScript modules. The features implemented in this project were:

- Created the src folder to host all the files necessary for the creation of the page: CSS files and js modules created.
- Created the dist folder to tell webpack where to store the final assets and files for the bundle run.
- Use DOM manipulation to create the content inside each navigation tab.
- User async/await with functions
- Use APIs
- Use classes to create a new object
- Loaders used from Webpack:
- style and css for CSS files.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @pastorp3 👋

**You've done quite well on this.👍**

**STATUS: CHANGES REQUESTED**👷‍♂️

- [x] OPTIONAL -  Consider fetching the geolocation of the user when the app is first accessed and then fetch the weather from that location, it would help to improve user experience.

- [x] Please refer to comments on your code.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/getdata.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  });
+}
+
+async function getData(input) {
+  try {
+    const request = await fetchData(input);
+    getImage(request.weather[0].main, request);
+  } catch (error) {
+    alert(error);
```
Comment (mikenath223) [inline src/getdata.js]: - [x] IMPORTANT - Please avoid using Window.alert in your code. Kindly remove this and all other cases where you added it. Handle errors in a better way. Window.alerts are ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/javascript-module-v-10/166782/path/step/97634928/))

**Comment by pastorp3** [comment]
Add review changes:
 - Delete window.alert()

**Review by bellom — APPROVED** [review]
## Hi @pastorp3  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**